Android/toonDra/app/build.gradle

```diff
@@ -28,4 +28,5 @@ dependencies {
   compile fileTree(include: ['*.jar'], dir: 'libs')
   compile 'com.android.support:appcompat-v7:24.2.1'
   testCompile 'junit:junit:4.12'
+  compile 'com.android.support:design:24.2.1'
```

Comment:
place this compile dependency line grouped along with other compile dependencies - above the testCompile dependency group